src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml

```diff
@@ -305,23 +314,115 @@
           <!-- the version column may be inserted here dynamically from code -->
 
           <!-- the installed column -->
-          <GridViewColumn
-            DisplayMemberBinding="{Binding InstalledVersion, Converter={StaticResource VersionToStringConverter}}">
+          <GridViewColumn>
             <GridViewColumn.Header>
               <GridViewColumnHeader
                 AutomationProperties.Name="{x:Static local:Resources.ColumnHeader_InstalledVersion}"
                 x:Name="_installedVersionColumnHeader"
                 Click="ColumnHeader_Clicked"
-                Content="{x:Static local:Resources.ColumnHeader_InstalledVersion}"
                 local:SortableColumnHeaderAttachedProperties.SortPropertyName="InstalledVersion"
                 HorizontalContentAlignment="Left"
                 SizeChanged="SortableColumnHeader_SizeChanged"
                 PreviewKeyUp="SortableColumnHeader_PreviewKeyUp"
                 Focusable="True"
                 IsTabStop="True"
+                FocusVisualStyle="{DynamicResource ControlsFocusVisualStyle}">
+                <StackPanel
+                  Orientation="Horizontal">
+                  <TextBlock
+                    Margin="2,0,0,0"
+                    VerticalAlignment="Bottom"
+                    Text="{x:Static local:Resources.ColumnHeader_InstalledVersion}" />
+                  <imaging:CrispImage
+                    Margin="2,0,0,0"
+                    x:Name="_warningIndicatorInstalledHeader"
+                    IsVisibleChanged="WarningIndicatorInstalledHeader_IsVisibleChanged"
+                    Visibility="{Binding InstalledVulnerabilitiesCount, Converter={StaticResource IsGreaterThanToVisibilityConverter}, ConverterParameter=0}"
+                    Moniker="{x:Static catalog:KnownMonikers.StatusWarning}">
+                    <imaging:CrispImage.ToolTip>
+                      <TextBlock>
+                        <TextBlock.Text>
+                          <MultiBinding Converter="{StaticResource StringFormatConverter}">
+                            <Binding Source="{x:Static local:Resources.Label_Installed_VulnerableWarning}" />
+                            <Binding Path="InstalledVulnerabilitiesCount" />
+                          </MultiBinding>
+                        </TextBlock.Text>
+                      </TextBlock>
+                    </imaging:CrispImage.ToolTip>
+                  </imaging:CrispImage>
+                </StackPanel>
+              </GridViewColumnHeader>
+            </GridViewColumn.Header>
+            <GridViewColumn.CellTemplate>
+              <DataTemplate>
+                <StackPanel
+                  VerticalAlignment="Bottom"
+                  Orientation="Horizontal">
+                  <TextBlock
+                    VerticalAlignment="Bottom"
+                    Text="{Binding InstalledVersion, Converter={StaticResource VersionToStringConverter}}" />
+                  <imaging:CrispImage
```

Comment:
When I presented this to the UX board the warning icons where on the right side but their suggestion was to move to the left side so it matches the Installed tab warning icon at the top

Reply:
@martinrrm do you mean vice versa? The icons are currently on the right side and I believe when you presented this initially you had them on the left and the advice was to put the warning icons on the right side. Please correct me if I'm wrong.

Reply:
Mb, I corrected the message, but yeah they suggested to change it so both installed "headers" have the icon at the same place, and have them mixed (rows with icon to the left and header to the right) would look weird

Reply:
Ok, the Installed tab could also have probably been changed. We still have 2 different approaches, again, the packages list shows on the left.

Reply:
Thanks Donnie, that a great suggestion, we have been getting UX approval for specific features and not considering the PM UI consistency. We should probably get back to the UX board and look at the whole PM UI. During my last UX meeting we discussed a lot of opportunities we could do improve the PM UI experience, I'll send a link to the meeting recording.

Reply:
Look forward to seeing that! In the meantime, if this is "ok" with UX board, then I think it could be easily changed later and not be too jarring to customers.

Reply:
Not for this PR, but could it make sense to move the vulnerability information to a column of its own? This would help align the icons and we could also add additional icons to denote the max severity of the vulnerabilities. 

✅ - No Vul
⚠️ - Low/Moderate/High
🔴 - Critical 
❔ - Unkown